src/main/java/ch/njol/skript/util/BlockLineIterator.java

```diff
@@ -87,14 +87,7 @@ public BlockLineIterator(Block start, Vector direction, double distance) throws
 	 * @return The newly modified Vector if needed.
 	 */
 	private static Vector fitInWorld(Location location, Vector direction) {
-		int lowest = WorldUtils.getWorldMinHeight(location.getWorld());
-		int highest = location.getWorld().getMaxHeight();
-		Vector vector = location.toVector();
-		int y = location.getBlockY();
-		if (y >= lowest && y <= highest)
-			return vector;
-		double newY = Math2.fit(lowest, location.getY(), highest);
-		return new Vector(location.getX(), newY, location.getZ());
+        return location.toVector();
```

Comment:
This method is now redundant, and the change makes it no longer do what it was intended to

Reply:
So are you saying to just completely delete the method?

Reply:
@TenFont

Reply:
is the method used somewhere else? if not, you can delete it. if it is, then don't change it